### DISCUSSION TITLE
How PyTensor is(will be) different from Aesara?

### DISCUSSION THREAD

**Discussion by apachaves** [pymc-devs/pytensor#9] [Q&A] [answered]
Hello, as a curious person over PyMC and Bayesian stats, I'm curious to know if someone can tell me where this new project inserts itself around the PyMC's ecosystem. Thanks in advance.

**Comment by michaelosthege** [comment]
Hi @apachaves, we appreciate the curiosity 🚀
There will be an official announcement in the next few days.

**Comment by michaelosthege** [comment, marked as answer]
The announcement is now online: https://www.pymc.io/blog/pytensor_announcement.html#pytensor_announcement

**Reply by apachaves** [reply]
Got it! Thank you, @michaelosthege!
